playbooks/roles/common-deployer/tasks/local-provisioner.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Clone local-path provisioner
+  git:
+    repo: "https://github.com/rancher/local-path-provisioner"
+    dest: "{{ upstream_repos_clone_folder }}/local-path-provisioner"
+    update: yes
+    force: yes
+    version: "v0.0.10"
```

Comment:
can this be sent in a var so we can test new versions by overriding it?

Reply:
I guess. Actually I only added the version because lint told me to